ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml

```diff
@@ -0,0 +1,576 @@
+        if [[ "$XTRACE_ENABLED" == 1 ]]; then
+          set -x
+        fi
+
+        mkdir -p /tmp/output
+        cd /tmp/output
+
+        function run-upgrade-tests() {
+          openshift-tests run-upgrade "${TEST_SUITE}" --to-image "${RELEASE_IMAGE_LATEST}" \
+            --provider "${TEST_PROVIDER:-}" -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
+          return 0
+        }
+
+        function run-tests() {
+          openshift-tests run "${TEST_SUITE}" \
+            --provider "${TEST_PROVIDER:-}" -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
+          return 0
+        }
+
+        function run-minimal-tests() {
+          # Only execute Smoke tests while the test infrastructure is getting
+          # prepared to run the actual suite reliably.
+          openshift-tests run openshift/conformance/parallel --dry-run |
+            grep 'Smoke' |
+          openshift-tests run -o /tmp/artifacts/e2e.log \
+            --junit-dir /tmp/artifacts/junit -f -
+          return 0
+        }
+
+        function run-no-tests() {
+          # This can be used if we just want to check the installer exits 0
+          echo "WARNING: No tests were run against the installed cluster"
+          return 0
+        }
+
+        ${TEST_COMMAND}
+
+    # Runs an install
+    - name: setup
+      image: ${IMAGE_OPENSTACK_INSTALLER}
+      terminationMessagePolicy: FallbackToLogsOnError
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      - name: cluster-profile
+        mountPath: /etc/openshift-installer
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: TYPE
+        value: ${CLUSTER_TYPE}
+      - name: CLUSTER_NAME
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      - name: BASE_DOMAIN
+        value: ${BASE_DOMAIN}
+      - name: SSH_PUB_KEY_PATH
+        value: /etc/openshift-installer/ssh-publickey
+      - name: SSH_PRIV_KEY_PATH
+        value: /etc/openshift-installer/ssh-privatekey
+      - name: PULL_SECRET_PATH
+        value: /etc/openshift-installer/pull-secret
+      - name: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+        value: ${RELEASE_IMAGE_LATEST}
+      - name: OPENSHIFT_INSTALL_INVOKER
+        value: openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}
+      - name: OPENSTACK_IMAGE
+        value: rhcos
+      - name: OPENSTACK_REGION
+        value: moc-kzn
+      - name: OPENSTACK_FLAVOR
+        value: m1.s2.xlarge
+      - name: OPENSTACK_EXTERNAL_NETWORK
+        value: external
+      - name: OS_CLOUD
+        value: openstack
+      - name: OS_CLIENT_CONFIG_FILE
+        value: /etc/openshift-installer/clouds.yaml
+      - name: USER
+        value: test
+      - name: HOME
+        value: /tmp
+      # we must boot from performance volumes to reduce disk speed related etcd failures
+      - name: OPENSTACK_MASTER_VOLUME_SIZE
+        value: "25"
+      - name: OPENSTACK_MASTER_VOLUME_TYPE
+        value: "performance"
```

Comment:
We're using these variables in the cluster-launch-installer-openstack-e2e.yaml file to allow the nodes to use more performant disks, using the boot from volume feature. Depending on your cloud, this trick may not be necessary.

https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md#machine-pools

Reply:
Removed for now, thanks!